### PR TITLE
Rename HostUtils.ExistsPath to PathExists

### PR DESCRIPTION
**Pull request by codenrhoden**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
As part of @thockin's review of https://github.com/kubernetes/utils/pull/100, this renaming was requested.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

/assign @msau42

### CODE REVIEW

File: pkg/kubelet/kubelet_pods.go [hunk 1]
```diff
@@ -32,7 +32,7 @@ import (
 	"strings"
 	"sync"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```
Comment (davidz627) [inline pkg/kubelet/kubelet_pods.go]: nit: unrelated change

File: pkg/util/mount/mount.go [hunk 2]
```diff
@@ -82,9 +82,9 @@ type HostUtils interface {
 	// MakeDir creates a new directory.
 	// Will operate in the host mount namespace if kubelet is running in a container
 	MakeDir(pathname string) error
-	// Will operate in the host mount namespace if kubelet is running in a container.
```
Comment (davidz627) [inline pkg/util/mount/mount.go]: do we still want this information in the comment?

Comment (msau42) [reply pkg/util/mount/mount.go]: We're preparing this package to be exported out to k8s/utils, and the review comments recommended removing K8s-specific information from the interface.

Although in this PR, you may just want to add a the new description to hopefully avoid merge conflicts with https://github.com/kubernetes/kubernetes/pull/81180/

File: pkg/kubelet/kubelet_pods.go (hunk 1 above)
Comment (msau42) [reply pkg/kubelet/kubelet_pods.go]: This is automatically modified by one of the go formatting tools. Many files in Kubernetes have this now.

**Review by msau42 — COMMENTED** [review]
/lgtm
/approve
/retest

**Review by mattjmcnaughton — APPROVED** [review]
/lgtm

Review lgtm on behalf of `pkg/kubelet/...`

/assign @saad-ali @tallclair

**Review by saad-ali — APPROVED** [review]
/lgtm
/approve
